src/renderer/screens/exchange/Swap2/Form/FormSelectors/FromRow.js

```diff
@@ -13,11 +13,11 @@ import Text from "~/renderer/components/Text";
 import { shallowAccountsSelector } from "~/renderer/reducers/accounts";
 import { amountInputContainerProps, selectRowStylesMap } from "./utils";
 import { FormLabel } from "./FormLabel";
-import type { Account } from "@ledgerhq/live-common/lib/types";
+import type { AccountLike } from "@ledgerhq/live-common/lib/types";
 
 type Props = {
-  fromAccount: ?Account,
-  setFromAccount: (?Account) => void,
+  fromAccount: ?AccountLike,
```

Comment:
You might want to deal with `account` and `parentAccount` 🤷🏼‍♂️ not a requirement though

Reply:
AccountLike is either `Account`, `TokenAccount` or `ChildAccount`.

```ts
// from *src/types/account.js*
export type SubAccount = TokenAccount | ChildAccount;
export type AccountLike = Account | SubAccount;
```

In another comment, you tell me to don't worry about ChildAccount so perhaps I can just use `Account` and `TokenAccount`?

```ts
type SwappableAccount = Account | TokenAccount;
```

Didn't find any types related to the parent accounts. Are `Account` and `TokenAccount` susceptible to be flagged as parents if they have got some child accounts?

Reply:
Parents are always top-level `Account`, so you will have something like this for a swap. 
```
const fromAccount: Account | TokenAccount;
const fromParentAccount?: Account;
```

Reply:
Thanks!